using-valtimo/document/custom-case-columns.md

```diff
@@ -0,0 +1,162 @@
+# Custom case columns
+
+Each case comes with a default case overview with four standard columns. 'Reference number', 'Created by', 'Created on',
```

Comment:
shouldn't we call it case list? i would personally read 'case overview ' as the detail page.